fragment.go

```diff
@@ -871,30 +947,41 @@ func (f *fragment) min(filter *Row, bitDepth uint) (min, count uint64, err error
 		return 0, 0, nil
 	}
 
-	for i := bitDepth; i > uint(0); i-- {
-		ii := i - 1 // allow for uint range: (bitDepth-1) to 0
-		row := f.row(uint64(ii))
+	// If we have negative values, we should find the highest unsigned value
+	// from that set, then negate it, and return it. For example, if values
+	// (-1, -2) exist, they are stored unsigned (1,2) with a negative sign bit
+	// set. We take the highest of that set (2) and negate it and return it.
+	if row := f.row(bsiSignBit).Intersect(consider); row.Count() != 0 {
```

Comment:
we have a row.Any() now.

it might make sense to add a row.IntersectAny(), for that matter, although that's a lot more work.